### PR TITLE
Fleshing out community landing page to address /website/issues/307

### PR DESCRIPTION
**Pull request by jessicaschilling**
See https://github.com/ipfs/website/issues/307 -- this is half the work in that issue.

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@ericronne, can you please have a look? Note that this leaves the IRC link at the bottom because that's Hugo's annoying auto-generated menu thing. Didn't dig deep into working around this, since it'll be a moot point in a few weeks in the new site.

File: content/community/_index.md [hunk 1]
```diff
@@ -3,3 +3,53 @@ title: "Community"
 headless: true
 opened: true
 ---
+
+The IPFS community is as decentralized and distributed as IPFS itself — our community members are globally located, active online and in person, and using and building on IPFS in a virtually endless number of ways. You can get more information about meetings, events, apps people are building, and more using the resources listed here.
```
Comment (ericronne) [inline content/community/_index.md]:
```suggestion
The IPFS community is as decentralized and distributed as IPFS itself. Our community members are globally located, active both online and in person, and use IPFS in countless different ways. You can learn about meetings, events, apps people are building, and more with the resources listed here.
```

File: content/community/_index.md [hunk 2]
```diff
@@ -3,3 +3,53 @@ title: "Community"
...
+
+The [IPFS Collaborative Notebook for Research](https://github.com/ipfs/notes) repo tracks discussions on topics related to IPFS research, ideas and proposals for new systems/features, and other items that haven't yet been tracked into a specific project.
+
+## Events Calendar
+
+Find all of the open community video calls for the IPFS project in the [IPFS Community events calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com).
+
+## Community Tools
+
+Join the discussion and documentation on IPFS community practices in the [community tools repo](https://github.com/ipfs/community/). If you're interested in starting, helping with or expanding IPFS awareness in your local geographical or online community, you can find tools and info here.
```
Comment (ericronne) [inline content/community/_index.md]: Not sure what "join the discussion and documentation" means here. Also lmk if my suggestion for the last sentence messed with its meaning!

```suggestion
Join the discussion and documentation on IPFS community practices in the [community tools repo](https://github.com/ipfs/community/). If you want to start or get involved with a community chapter to help expand IPFS awareness in your local geographical or online community, here's where to look.
```

File: content/community/_index.md [hunk 3]
```diff
@@ -3,3 +3,53 @@ title: "Community"
...
+
+## Project Management
+
+Keep abreast on IPFS team planning, management and coordination issues in the [IPFS team management repo](https://github.com/ipfs/team-mgmt).
```
Comment (ericronne) [inline content/community/_index.md]:
```suggestion
Keep abreast of IPFS team planning, management and coordination issues in the [IPFS team management repo](https://github.com/ipfs/team-mgmt).
```

File: content/community/_index.md [hunk 4]
```diff
@@ -3,3 +3,53 @@ title: "Community"
...
+
+## Blog
+
+Explore the latest news, events and other happenings in the IPFS multiverse in the official [https://ipfs.io/blog/](IPFS Blog).
```
Comment (ericronne) [inline content/community/_index.md]:
```suggestion
Explore the latest news, events and other happenings in the IPFS multiverse on the official [https://ipfs.io/blog/](IPFS Blog).
```

File: content/community/_index.md [hunk 5]
```diff
@@ -3,3 +3,53 @@ title: "Community"
...
+
+Get your IPFS news in tweet-sized bites. Follow these accounts for the latest:
+
+- [@IPFSbot](https://twitter.com/IPFSbot) for news and other updates
+- [@IPFSevents](https://twitter.com/ipfsevents) for info on IPFS-related events
+- [@ProtoSchool](https://twitter.com/protoschool) for updates on ProtoSchool chapters and tutorials
+
+## Community projects on Awesome IPFS
+
+Get inspired by [Awesome IPFS](https://awesome.ipfs.io/), a catalog of community-built projects using IPFS for everything from image sharing to decentralized archiving. Have a project of your own to add to Awesome IPFS? Make a PR in the [repo](https://github.com/ipfs/awesome-ipfs)!
```
Comment (ericronne) [inline content/community/_index.md]:
```suggestion
Get inspired by [Awesome IPFS](https://awesome.ipfs.io/), a catalog of community-built projects using IPFS for everything from image sharing to decentralized archiving. Have a project of your own to add to Awesome IPFS? [Make a PR](https://github.com/ipfs/awesome-ipfs) in the repo!
```

**Comment by jessicaschilling** [comment]
Thanks!

**Pull request merged**